examples/controls/src/app.js

```diff
@@ -18,28 +18,24 @@ import CITIES from '../../.data/cities.json';
 const TOKEN = ''; // Set your mapbox token here
 
 const geolocateStyle = {
-  position: 'absolute',
```

Comment:
was position absolute not useful in these cases?

Reply:
We already merge user-provided style with `{position: 'absolute'}`